src/api/ledger/parse/utils.js

```diff
@@ -62,8 +62,9 @@ function parseOutcome(tx: Object): ?Object {
     fee: utils.common.dropsToXrp(tx.Fee),
     balanceChanges: balanceChanges,
     orderbookChanges: orderbookChanges,
-    ledgerVersion: tx.ledger_index,
-    indexInLedger: tx.meta.TransactionIndex
+    ledgerVersion: typeof tx.ledger_index === 'number' ? tx.ledger_index : 0,
```

Comment:
I've only seen 'ledger_index' as a string in the ledger header (i.e.
'ledger' cmd responses) but there might be others

@rec is looking at making it consistently a number